package-tracking/urls.py

```diff
@@ -20,10 +20,10 @@
     path('contact/', contact_view),
     path('financials/', financial_view),
 
-    path('partners/<str:partner_name>/', partner_view),
-    path('partners/<str:partner_name>/<str:order_id>/', order_edit_view),
-    path('packages/<str:package_id>/', package_view),
-    path('orders/<str:order_id>/', order_view),
-    path('staff/', staff_view),
+    path('partner/<str:partner_name>/', partner_view),
+    path('partner/<str:partner_name>/<str:order_id>/', order_edit_view),
+    path('package/<str:package_id>/', package_view),
+    path('order/<str:order_id>/', order_view),
+    path('partners/', partners_view),

```

Comment:
You can also do `partner/`, showing an hierarchal relationship with the other partner paths.

Reply:
good idea

Reply:
It can be `partner/` and that's it. Not need for `partner/partners`

Reply:
oh, without the id?

Reply:
ok changed it

Reply:
Did you? Because it doesn't seem like you pushed it.